ethereumj-core/src/main/java/org/ethereum/exception/FatalEthereumException.java

```diff
@@ -0,0 +1,21 @@
+package org.ethereum.exception;
```

Comment:
So this one and its parent is not used anymore (and its parent)
Do we need to keep it?
And also copyrights are missed both for this one and its parent.

Reply:
Yes, we don't use it anymore in this case, but it can be useful in the future to throw an exception that could shutdown an app.

Reply:
I'd keep it simple and leave only `FatalBeanException` for the moment.